easybuild/easyconfigs/p/Python/Python-2.7.9-gompi-1.5.16.eb

```diff
@@ -0,0 +1,24 @@
+name = 'Python'
+version = '2.7.9'
+
```

Comment:
I have no ideological problem with it, just never saw it before. I used other PR Python configs as template. Never saw -bare version suffix between released ebfiles. 

Is this something common down the road? Will it be applied in the future releases to other configs?